dev-util/rebar3/rebar3-3.6.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit bash-completion-r1
+
+DESCRIPTION="A sophisticated build-tool for Erlang projects that follows OTP principles"
+HOMEPAGE="https://github.com/erlang/rebar3"
+SRC_URI="https://github.com/erlang/rebar3/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ia64 ~ppc ppc64 ~sparc ~x86"
+IUSE="bash-completion"
```

Comment:
the flag is not needed

Reply:
Why not make use of an established "bash-completion" flag, like the other 219 packages in the main tree that already do that?

Reply:
we do not ship tiny files flagged anymore, a single completion file does not eat your diskspace but only makes you to perform needless rebuilds

Reply:
Of course you do: ```ag -l bash-completion /usr/portage | cut -d '/' -f 1-5 | uniq | wc -l```
The rationale is that people not using Bash or not needing completion for it, don't have to install all these small files.

Reply:
There is the `INSTALL_MASK` variable for those who do not want to install some specific files (thats how systemd files getting abolished in the end systems currently).

Reply:
Asking the user to replace a documented global USE flag with an undocumented path where files should not be installed is a step back in usability. This is an avoidable and very ugly hack.

Stick to USE flags for controlling what gets installed and what doesn't and you'll avoid surprising the user.

Reply:
That's the consensus. The gentoo tree follows rules that are built around a consensus. Whether you agree with the consensus or not is irrelevant: the ebuild has to follow it to be merged. These rules apply to every PR and every commit.

Reply:
Sorry, but the Gentoo of 2018 tries to keep exponential combinatorial explosion for 2 kB installed files to a minimum. Rest assured, the remaining ebuilds will be changed too.